### ISSUE TITLE
Future of migrate

### ISSUE THREAD

**Issue by mattes** [mattes/migrate#311]
Everyone,

I started migrate 3 years ago and I'm happy it's seeing widespread adoption. I see forks at @99designs, @codeship, @uber, @segmentio, @newrelic, @gitgnu, @postmates and many more who find migrate helpful and try to help pushing it forward.

Some months ago I co-founded www.templarbit.com and ever since things are busier than ever. It's unfortunate, but I won't be able to keep maintaining migrate.

Who is interested in stepping in and taking migrate over?

**Comment by jadbox** [comment]
I just discovered this tool.. and sad that it's already going to lose its author. 

Would you be willing to move the repo to its own org (maybe 'gomigrate') and allow active contributors to have commit access? It's probably the best way to keep the project maintained by giving ownership over to the lead contributors.

**Comment by dhui** [comment]
I've created a [fork](https://github.com/golang-migrate/migrate) with the following goals:
1. Get tests passing again
1. Merge in existing open PRs that add value (this is subjective) and require very little effort to verify correctness
1. Push a new release

I won't have much time to dedicate to maintaining this fork so I'm looking for co-maintainers as well.

**Comment by cactus** [comment]
@dhui looks like the build is broken due to cockroachdb not supporting go1.7 (it apparently uses a few things that apparently were added to go1.8).

```
# github.com/cockroachdb/cockroach-go/crdb
../../cockroachdb/cockroach-go/crdb/tx.go:41: undefined: sql.TxOptions
../../cockroachdb/cockroach-go/crdb/tx.go:43: db.BeginTx undefined (type *sql.DB has no field or method BeginTx)
```

**Comment by dhui** [comment]
FYI, the fork is now ready to use!

https://github.com/golang-migrate/migrate/releases/tag/v3.1.0

Tests are passing again, so contributions are welcome.

**Comment by dcormier** [comment]
It would be nice to see additional committers in that fork to avoid this happening again, @dhui.

**Comment by dhui** [comment]
@dcormier

> It would be nice to see additional committers in that fork to avoid this happening again, @dhui.

Agreed, but that's beyond my control. Technically you're a committer since I cherry-picked your commit 😉 

Maybe we'll have another PR merged in after this one was closed: https://github.com/golang-migrate/migrate/pull/4

**Comment by dcormier** [comment]
> Technically you're a committer since I cherry-picked your commit

I guess I mean someone else (anyone) who is responsive and has control of the org, in case you're kidnapped by pygmies.

**Comment by avelino** [comment]
@mattes and maintainer team
We use a lot of the migrate in pREST[1], if you want to move the project to the organization of pREST[2] will be very welcome (we have a great team to keep it)

[1] https://github.com/prest/prest
[2] https://github.com/prest

**Comment by dhui** [comment]
@avelino I'm hesitant to move the repo again unless there's a good reason to do so.

You and your team should feel free to make contributions to the [fork](https://github.com/golang-migrate/migrate). I can also make you and your team collaborators if you so desire once you've made a few contributions.

**Comment by mixedCase** [comment]
@mattes Could a fork be blessed as the official continuation and this repo marked in the README as abandoned? I vendored it and was only aware it had been abandoned because I needed the Cassandra driver and it wasn't available in the latest release.

And now's a good a time as any: Thank you for all the work you did on migrate, it has been my go-to choice for Go projects for a while and certainly proved useful. Best of luck in your new endeavor!

**Comment by nfisher** [comment]
@dhui what're your thoughts on just adding @avelino and @dcormier as admins? Worst case they do nothing... best case they help a little. :)
